app/assets/stylesheets/content/_forms.sass

```diff
@@ -532,6 +532,12 @@ fieldset.form--fieldset
     background: none
     border: none
 
+.form--tooltip-container
+  flex: 0 0 auto
+  display: flex
+  span
```

Comment:
@floriank I'd prefer to see a class selector here (or list of classes). Even better, `& > .tooltip-right` if it should always apply to a direct descendent.